GMnetENGINE.gmx/objects/htme_obj_waitforclient.object.gmx

```diff
@@ -31,7 +31,8 @@ if (htme_clientIsConnected()) {
 }
 if (htme_clientConnectionFailed()) {
     show_message("Connection with server failed!");
-    game_restart();
+    htme_clientStop();

```

Comment:
Clean client when connection failed and go to a room. The engine will re-init itself in the next room.